clients/src/test/java/org/apache/kafka/common/network/CertStores.java

```diff
@@ -41,23 +41,23 @@
 
     private final Map<String, Object> sslConfig;
 
-    public CertStores(boolean server, String hostName) throws Exception {
-        this(server, hostName, new TestSslUtils.CertificateBuilder());
+    public CertStores(boolean server, String hostName, boolean usingOpenSsl) throws Exception {
```

Comment:
can you document the purpose of this new flag?

Reply:
also, may be worth introducing an enum to make similar efforts by others easier ?

Reply:
Add an enum in `TestSslUtils`